readme.md

```diff
@@ -209,8 +209,7 @@ A basic walkthrough is outlined in the structure below:
 ```
 @kirbydesign/designsystem
 ├── apps                    # Contains source code for applications
-|  ├── cookbook             # - Cookbook application (showcase and examples)
-|  └── cookbook-e2e         # - End-to-end tests for Cookbook application
+|  └── cookbook             # - Cookbook application (showcase and examples)
 ├── config
 |  └── helm
```

Comment:
We can remove helm here too, right?

Reply:
![](https://c.tenor.com/IgXh2WY_WCEAAAAC/mabel-gravity-falls.gif)